massgov/mayflower-patternlab
# DP-7265 mosaic grid

## Description

Add Featured Item molecule and Featured Item Mosaic organism for initial use on the elected official page type.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-7265)

## Steps to Test


1. Check out the invision link: https://invis.io/VYF2N5JX2
2. Ensure that things look right at: http://localhost:3000/?p=organisms-featured-item-mosaic

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* new molecule @molecules/featured-item
* new organism @organisms/featured-item-mosaic
* new page @pages/organization-elected-official

#### @TODO


* I believe I need to add a wrapping block somewhere so Drupal can override, though I'm not sure exactly where that would be most useful here.

#### Today I learned...
